components/ConnectWallet/ConnectWallet.tsx

```diff
@@ -11,7 +11,12 @@ export const ConnectWallet = () => {
     <ConnectButton.Custom>
       {({ account, openConnectModal }) =>
         !account ? (
-          <Button onClick={openConnectModal} type="button">
+          <Button
+            onClick={() => {
+              window.pirsch('Wallet connection modal opened', {});
```

Comment:
unfortunately I don't think we have a great way to measure the conversion rate here, we don't get like an `onConnect` callback and other ideas I have may be prone to double-counting.